apps/jobs/serializers.py

```diff
@@ -32,7 +32,9 @@ class Meta:
         fields = ('id', 'participant_team', 'participant_team_name', 'execution_time', 'challenge_phase',
                   'created_by', 'status', 'input_file', 'stdout_file', 'stderr_file', 'submitted_at',
                   'method_name', 'method_description', 'project_url', 'publication_url', 'is_public',
-                  'submission_result_file',)
+                  'submission_result_file', 'submissions_count_on_a_challenge_phase',
```

Comment:
This is something which I am totally missing. From my understanding, we should provide these property value in challenge phase analysis, not whenever any one is making a submission.

With the present change in PR, any making a submission will be actually able to see all these stats.

@deshraj